CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ $ npm run test
 
 ### _We actively welcome your pull requests, however linking your work to an existing issue is preferred._
 
-1. Fork the repo and create your branch from `master`.
+1. Fork the repo and create your branch from `main`.
 1. Name your branch something that is descriptive to the work you are doing. i.e. adds-new-thing or fixes-modbile-render
```

Comment:
You can slide in a quick typo fix in this PR as well.
`fixes-modbile-render` to `fixes-mobile-render`